swapnil/online_photo_system/photos.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en-US">
+    <head>
+        <title>Photos</title>
+        <meta name="description" content="This contain information about Online photography system">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+    <body>
+        <div class="container">
+            <div class="nav">
+                <ul>
+                    <li><a href="index.html">Home</a></li>
+                    <li><a class="active" href="photos.html">Photos</a></li>
+                    <li><a href="bio.html">Bio</a></li>
+                    <li><a href="services.html">Services</a></li>
+                    <li><a href="contact.html">Contact</a></li>
+                </ul>
+            </div>
+            <div class="picture">
+                <a href="pic.html" target=_blank>
+                <img src="images/hummer.jpg" height="500" width="500" style="padding-right:20px" alt="hummer" > 
+                <img src="images/creative.jpg" height="500" width="800" alt="creative">
+                <img src="images/aqua.jpg" height="500" width="100%" alt="aqua">
+                <img src="images/bird.jpg" height="500" width="500" style="padding-right:20px" alt="bird">
+                <img src="images/flower.jpg" height="500" width="800" alt="flower">
+                <img src="images/animal.jpg" height="500" width="800"style="padding-right:20px" alt="animal">
+                <img src="images/tech.jpg" height="500" width="500" alt="tech">
+                <img src="images/nature.jpg" height="500" width="100%" alt="nature">
+                <img src="images/wonder.jpg" height="500" width="100%" alt="wonder">
+                </a>
+            </div>
+            <div class="footer">
+                <p>All Rights Reserved By Swapnil Sangvikar </p>
+            </div>
+        </div>
+    </body>
+</html>
```

Comment:
add more contents for footer 
add more css properties for this  page
need to add  name for every single photo 
need proper photo alignment